3rd_party/3rd_party.sh

```diff
@@ -68,6 +69,8 @@ case `uname` in
             XML_EXTENSION=.so.2
             GCC_RT_LOCATION=/usr/local/gcc93/lib64
             GCC_RT_EXTENSION=.so.1
+            OMP_LOCATION=/usr/local/gcc93/lib64
```

Comment:
I'm guessing this is installed by the Python dependencies as I don't see it explicitly built in the dockerfile

Reply:
`libgomp` is part of gcc and built alongside it, like `libgcc_s` and `libstdc++`.